python/afdko/convertfonttocid.py

```diff
@@ -4,7 +4,7 @@
 convertfonttocid.py. v 1.13.1 Jul 12 2019
 
 Convert a Type 1 font to CID, given multiple hint dict defs in the
-"fontinfo" file. See autohint help, with the "-hfd" option, or the makeotf
+"fontinfo" file. See psautohint help, with the "-hfd" option, or the makeotf
```

Comment:
`-hfd` → `--doc-fddict`
BTW, I didn't get a deprecation message in the console when I used the legacy option.

Reply:
Probably [this](https://github.com/adobe-type-tools/psautohint/blob/d1ca70affebe4f070037365d44780ab8a78778b0/python/psautohint/__main__.py#L829) is the cause of the deprecation message not appearing -- it doesn't use `_DeprecatedAction` like the others...possibly because it uses an extended help. I opened https://github.com/adobe-type-tools/psautohint/issues/260 to track. I do not think it is an urgent fix.

Reply:
flag descriptor fixed in https://github.com/adobe-type-tools/afdko/pull/1175/commits/567b665afd6d67927896b18f29ceec99c7b6c124